portfolio.css

```diff
@@ -453,9 +453,9 @@ h3 {
   background-image: url(images/Frame.png);
   background-repeat: no-repeat;
   background-attachment: scroll;
-  background-position: 83% 32%;
+  background-position: 83% 28%;
```

Comment:
- [ ] Your CSS is not correctly  working for the mobile view see screenshot.

- Your project 👇.
![jkjkjkj](https://user-images.githubusercontent.com/61562626/134188506-3b679c5c-fdbc-4806-a711-9a42bf7d7286.jpeg)

- Original project 👇

![jjj](https://user-images.githubusercontent.com/61562626/134189674-c2cf53bd-28ef-4ca2-981e-a66b864d45ec.gif)

Reply:
Kindly work on the changes requested by your last reviewer. 
Stills missing the design provided on Figma
![Screen Shot 2021-09-21 at 11 06 24](https://user-images.githubusercontent.com/22258274/134206868-3bca512b-29ea-4564-9976-5f877937866c.png)
![Screen Shot 2021-09-21 at 11 06 40](https://user-images.githubusercontent.com/22258274/134206869-600cbdb9-aaf0-4353-becd-8b7388cbf613.png)
.

Before to request another code review ensure to have all the features requested on your project requirements.

Reply:
Hello. I understand the problem with the navbar, I increased the width to the size of the body including the margin of the body. But I do not understand the second problem (the second picture). This is how it looks on my pc.
![Review problem](https://user-images.githubusercontent.com/79739022/134210438-043a2c59-a2c5-4408-85e6-82df5a460ff2.png)
[](url)